jmclient/jmclient/payjoin.py

```diff
@@ -745,9 +745,8 @@ def request_to_psbt(self, payment_psbt_base64, sender_parameters):
         # it is still safer to at least verify the validity of the signatures
         # at this stage, to ensure no misbehaviour with using inputs
         # that are not signed correctly:
-        res = jm_single().bc_interface.testmempoolaccept(bintohex(
-            self.manager.payment_tx.serialize()))
-        if not res[0]["allowed"]:
+        if not jm_single().bc_interface.testmempoolaccept(bintohex(
+                self.manager.payment_tx.serialize())):
```

Comment:
But this changes the logic? The former checks a field allowed at list index 1?

Reply:
No, I changed `testmempoolaccept()` to return `bool` instead of all data, as we only used `res[0]["allowed"]` everywhere.